wpuf-functions.php

```diff
@@ -4055,6 +4013,23 @@ function wpuf_payment_success_page( $data ){
     return $redirect_page;
 }
 
+/**
+ *  Inconsistency with keys, remap keys, Back compat with keys
+ *
+ * @param $address_fields
+ * @return array
+ */
+function wpuf_get_address( $address_fields ) {
```

Comment:
The function name suggests it gets the address from somewhere, like a user or from database. But in reality, it maps the field and returns. It could be something like `wpuf_map_address_fields`.

Reply:
I've named it wpuf_billing_address_remap and after your suggestion, it renames it to wpuf_get_address. will rename as you suggest.